src/library/Uno.Toolkit.Material/Styles/Controls/v2/BottomTabBar.xaml

```diff
@@ -0,0 +1,707 @@
+
+	<StaticResource x:Key="M3MaterialBottomTabBarItemIconForeground"
+	                ResourceKey="OnSurfaceVariantBrush" />
+	<StaticResource x:Key="M3MaterialBottomTabBarItemIconForegroundSelected"
+	                ResourceKey="OnSecondaryContainerBrush" />
+
+
+	<StaticResource x:Key="M3MaterialBottomTabBarItemActiveIndicatorBackgroundSelected"
+	                ResourceKey="SecondaryContainerBrush" />
+
+	<SolidColorBrush x:Key="M3MaterialBottomTabBarBackgroundTint"
+	                 Opacity="0.08"
+	                 Color="{ThemeResource PrimaryColor}" />
+
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemDisabledBackground"
+	                ResourceKey="SystemControlTransparentBrush" />
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemDisabledForeground"
+	                ResourceKey="OnSurfaceDisabledBrush" />
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemPointerOverStateOverlay"
+	                ResourceKey="OnPrimaryContainerHoverBrush" />
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemFocusedStateOverlay"
+	                ResourceKey="OnPrimaryContainerFocusedBrush" />
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemPressedStateOverlay"
+	                ResourceKey="OnPrimaryContainerPressedBrush" />
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemDisabledStateOverlay"
+	                ResourceKey="OnSurfaceDisabledLowBrush" />
+
+	<x:Double x:Key="M3MaterialBottomTabBarFontSize">12</x:Double>
+	<FontWeight x:Key="M3MaterialBottomTabBarFontWeight">Medium</FontWeight>
+	<x:Double x:Key="M3MaterialBottomTabBarHeight">80</x:Double>
+	<GridLength x:Key="M3MaterialBottomTabBarGridLengthHeight">80</GridLength>
+	<x:Double x:Key="FabItemVerticalOffset">-32</x:Double>
+	<x:Double x:Key="M3MaterialBottomTabBarItemIconHeight">16</x:Double>
+	<x:Double x:Key="M3MaterialBottomTabBarItemIconWidth">16</x:Double>
+	<Thickness x:Key="M3MaterialBottomTabBarItemContentMargin">0,0,0,6</Thickness>
+	<Thickness x:Key="M3MaterialBottomTabBarItemContentOnlyMargin">12,0</Thickness>
+	<CornerRadius x:Key="M3MaterialBottomTabBarItemActiveIndicatorCornerRadius">16</CornerRadius>
+	<Thickness x:Key="M3MaterialBottomTabBarItemActiveIndicatorPadding">26,10</Thickness>
+	<!--  Code can be removed when this issue is fixed in Uno:  -->
+	<!--  https://github.com/unoplatform/uno/issues/7393  -->
+	<!--  AjustedMaterialBottomTabBarHeight = MaterialBottomTabBarHeight + Half size of the Fab TabBarItem (including ElevatedView margin)  -->
+	<x:Double x:Key="AjustedMaterialBottomTabBarHeight">108</x:Double>
+	<Thickness x:Key="M3MaterialBottomFabTabBarItemPadding">16</Thickness>
+	<x:Double x:Key="M3MaterialBottomFabTabBarItemContentWidthOrHeight">18</x:Double>
+	<CornerRadius x:Key="M3MaterialBottomFabTabBarItemCornerRadius">16</CornerRadius>
+	<x:Double x:Key="M3MaterialBottomFabTabBarItemIconTextPadding">12</x:Double>
+
+
+	<!--  Material Bottom TabBar  -->
+	<mobile:Style x:Key="M3MaterialBottomTabBarStyle"
```

Comment:
i see multiple set of ControlTemplate in this file, it would be nice at the beginning of the xaml file
to quickly list them and give an overview of shared/non-shared parts
it would be helpful to the next person or you in couple days/weeks working in this file too